packages/editor-ui/src/stores/ui.store.ts

```diff
@@ -87,545 +81,660 @@ try {
-		},
-		getModalMode() {
-			return (name: ModalKey) => this.modals[name].mode;
-		},
-		getModalData() {
-			return (name: ModalKey) => this.modals[name].data;
-		},
-		getFakeDoorByLocation() {
-			return (location: IFakeDoorLocation) =>
-				this.fakeDoorFeatures.filter((fakeDoor) => fakeDoor.uiLocations.includes(location));
-		},
-		getFakeDoorById() {
-			return (id: string) =>
-				this.fakeDoorFeatures.find((fakeDoor) => fakeDoor.id.toString() === id);
-		},
-		isReadOnlyView(): boolean {
-			return ![VIEWS.WORKFLOW, VIEWS.NEW_WORKFLOW, VIEWS.EXECUTION_DEBUG].includes(
-				this.currentView as VIEWS,
-			);
-		},
-		isNodeView(): boolean {
-			return [
-				VIEWS.NEW_WORKFLOW.toString(),
-				VIEWS.WORKFLOW.toString(),
-				VIEWS.WORKFLOW_EXECUTIONS.toString(),
-			].includes(this.currentView);
-		},
-		isActionActive() {
-			return (action: string) => this.activeActions.includes(action);
-		},
-		getSelectedNodes(): INodeUi[] {
-			const seen = new Set();
-			return this.selectedNodes.filter((node: INodeUi) => {
-				// dedupe for instances when same node is selected in different ways
-				if (!seen.has(node.id)) {
-					seen.add(node.id);
+			return false;
+		});
+	});
+
+	const isNodeSelected = computed(() => {
+		return (nodeName: string): boolean => {
+			for (const [index] of selectedNodes.value.entries()) {
+				if (selectedNodes.value[index].name === nodeName) {
 					return true;
 				}
-				return false;
-			});
-		},
-		isNodeSelected() {
-			return (nodeName: string): boolean => {
-				let index;
-				for (index in this.selectedNodes) {
-					if (this.selectedNodes[index].name === nodeName) {
-						return true;
-					}
-				}
-				return false;
-			};
-		},
-		upgradeLinkUrl() {
-			return async (source: string, utm_campaign: string, deploymentType: string) => {
-				let linkUrl = '';
-
-				const searchParams = new URLSearchParams();
-
-				if (deploymentType === 'cloud' && hasPermission(['instanceOwner'])) {
-					const adminPanelHost = new URL(window.location.href).host.split('.').slice(1).join('.');
-					const { code } = await useCloudPlanStore().getAutoLoginCode();
-					linkUrl = `https://${adminPanelHost}/login`;
-					searchParams.set('code', code);
-					searchParams.set('returnPath', '/account/change-plan');
-				} else {
-					linkUrl = N8N_PRICING_PAGE_URL;
-				}
+			}
+			return false;
+		};
+	});
 
-				if (utm_campaign) {
-					searchParams.set('utm_campaign', utm_campaign);
-				}
+	const upgradeLinkUrl = computed(() => {
+		return async (source: string, utm_campaign: string, deploymentType: string) => {
+			let linkUrl = '';
 
-				if (source) {
-					searchParams.set('source', source);
-				}
-				return `${linkUrl}?${searchParams.toString()}`;
-			};
-		},
-		headerHeight() {
-			const style = getComputedStyle(document.body);
-			return Number(style.getPropertyValue('--header-height'));
-		},
-		isAnyModalOpen(): boolean {
-			return this.modalStack.length > 0;
-		},
```

Comment:
This computed property seems to be used in multiple places. I remember using it for the pin data discovery tooltip. I don't think it's safe to remove.

Reply:
Huge overlooked. I must have had the wrong filters in the search and got misled into thinking it was not used anymore. Awesome catch. Thanks